src/connect/order/isOldOrder.ts

```diff
@@ -0,0 +1,3 @@
+export function isOldOrder(orderId: string): boolean {
```

Comment:
I think we can give this a more descriptive name: [see php-sdk](https://github.com/paynl/php-sdk/blob/9627b58ae8cdfa5cceaf61fb8d89756b9e73dc06/src/Util/Misc.php#L172)

```suggestion
export function isNotTguOrder(orderId: string): boolean {
```

Reply:
Sure, although I find it doubtful that your suggestion is more descriptive.